src/picode/commandline/LionWeb2FreonTemplate.ts

```diff
@@ -0,0 +1,64 @@
+import { FreModelUnit } from "@freon4dsl/core";
+import { Concept, ConceptInterface, Containment, Enumeration, Feature, Language, PrimitiveType, Property, Reference } from "../language/gen/index";
+
+export class LionWeb2FreonTemplate {
+
+    generateFreonAst(metamodel: FreModelUnit): string {
+        let result = "";
+        result += (`language ${metamodel.name}\n`);
+        (metamodel as Language).entities.forEach(entity => {
+            switch(entity.freLanguageConcept()) {
+                case "Concept":
```

Comment:
(I guess a case for `Annotation` is missing here.)

Reply:
yes, this code is WIP

Reply:
Changed it

Reply:
I ned to addd Annotation support in Freon first :-(.